_data/committers.yml

```
@@ -244,6 +244,10 @@
   role: Committer
   alias: icexelloss
   affiliation: Two Sigma
+- name: Liang-Chi Hsieh
+  role: Committer
+  alias: viirya
+  affiliation: TBD
```

```suggestion
  affiliation: Apple
```

Added. Thanks @alamb